python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/monitor-sessions.j2

```diff
@@ -54,10 +54,19 @@
 {%             endif %}
 {%         endif %}
 {%     endfor %}
+{%     if monitor_sessions_default_encapsulation_gre.payload is arista.avd.defined %}
+
+### Monitor Sessions Default
```

Comment:
I tried this:- 

Remove this logic. Create a new documentation file with name monitor-session-default-encapsulation-gre.j2, add the  same logic and this template in documentation/monitoring.j2. Then it works even if only monitor_session_default_encapusation key is defined in host.